packages/clay-drop-down/src/Item.tsx

```diff
@@ -75,6 +75,7 @@ const ClayDropDownItem: React.FunctionComponent<IProps> = ({
 				href={href}
 				onClick={onClick}
 				ref={innerRef}
+				tabIndex={disabled ? -1 : otherProps.tabIndex}
```

Comment:
Instead of pulling tabIndex from otherProps, lets just destructure tabIndex from above

Reply:
Good!